handbook/company/why-this-way.md

```diff
@@ -72,5 +72,12 @@ Groups are organized around goals. Connecting people with the same goals helps t
 
 Every group at Fleet maintains specific Slack channels, which all group members join and keep unmuted.  Everyone else at Fleet is encouraged to mute these channels, using them only as needed.  Each channel has a directly responsible individual responsible for keeping up with all new messages, even if they aren't explicitly mentioned (`@`).
 
+## Why does Fleet value Openness?
+Q: "Hey Mike, what are some of the pros and cons of a company making information public by default?"
```

Comment:
I have some revisions that are a bit more extensive, but I think get at the heart of the matter. If you and Mike agree, can you please delete this line?